terraform/main.tf

```diff
@@ -23,18 +23,20 @@ resource "google_compute_subnetwork" "honeypot_subnet" {
   network       = google_compute_network.main.self_link
 }
 
-# standard firewall settings
-resource "google_compute_firewall" "firewall_standard_rule" {
-  name    = "firewall-standard-rule"
+resource "google_compute_firewall" "containerssh-allow-all" {
```

Comment:
release all firewall rules for easier development. we could set it up later when everything works